packages/plugin-hardhat/src/index.ts

```diff
@@ -69,7 +69,9 @@ interface RunCompilerArgs {
 }
 
 subtask(TASK_COMPILE_SOLIDITY, async (args: { force: boolean }, hre, runSuper) => {
-  const { readValidations, ValidationsCacheOutdated, ValidationsCacheNotFound } = await import('./utils/validations');
+  const { readValidations, ValidationsCacheOutdated, ValidationsCacheNotFound } = await import(
+    './utils/validations.js'
```

Comment:
Why are these `.js` required now?

Reply:
This is because `moduleResolution` was changed to `nodenext` in tsconfig.base.json, in order to support importing Slang.

Omitting the `.js` extension here causes error `Relative import paths need explicit file extensions in ECMAScript imports when '--moduleResolution' is 'node16' or 'nodenext'.`